src/lua/LuaPiGui.cpp

```diff
@@ -1235,7 +1235,7 @@ static int l_pigui_add_bezier_curve(lua_State *l)
 	ImColor color = LuaPull<ImColor>(l, 5);
 	float thickness = LuaPull<double>(l, 6);
 	int num_segments = LuaPull<int>(l, 7);
-	draw_list->AddBezierCurve(a, c0, c1, b, color, thickness, num_segments);
+	draw_list->AddBezierCubic(a, c0, c1, b, color, thickness, num_segments);
```

Comment:
In this case Bezier Curve is the correct name, Cubic and Bezier curves are different things

Reply:
The curves added here are in fact cubic Bezier curves, in contrast to for example quadratic Bezier curves - the recent naming changes in imgui reflect that and make the difference clearer.
If you look at [line 2491 in imgui.h](https://github.com/ocornut/imgui/blob/607ad8c949a5411f4ce069e8e0f8ae773718e7ec/imgui.h#L2491), you can see that AddBezierCurve() is nothing more than a redirect to AddBezierCubic() and will be removed entirely in the near future. See also the linked commits above.

Reply:
Ah ok